apps/condo/domains/ticket/schema/fields/TicketOrganizationSetting.js

```diff
@@ -0,0 +1,57 @@
+const { Integer } = require('@keystonejs/fields')
```

Comment:
it looks like you need to remove all this tricky runtime generation logic. Just define all fields directly inside the schema. It's easier to read. Think about team, it's hard to read ...